pkg/common/security/usergroup_test.go

```diff
@@ -64,27 +109,29 @@ func TestGetUserGroup(t *testing.T) {
 	}
 	// click over the clock: if we do not get the cached version the new time will differ from the cache update
 	cachedUG.resolved -= 5
+	testCache.lock.Unlock()
+
 	ug, err = testCache.GetUserGroup("testuser1")
+	testCache.lock.Lock()
 	if err != nil || ug.resolved != cachedUG.resolved {
 		t.Errorf("User 'testuser1' not returned from Cache, resolution time differs: %d got %d (err = %v)", ug.resolved, cachedUG.resolved, err)
 	}
+	testCache.lock.Unlock()
```

Comment:
This lock can be removed. `GetUserGroup()` returns a copy and internally uses a mutex to access the map.